vllm/spec_decode/spec_decode_worker.py

```diff
@@ -127,7 +125,6 @@ def create_worker(
 
             if draft_worker_kwargs[
                     "model_config"].hf_config.model_type == "mlp_speculator":
-                disable_bonus_tokens = False
```

Comment:
can we either remove the `disable_bonus_token` paths entirely, or log below whether the bonus token is enabled?

Reply:
Removed disable_bonus_token.